packages/runtime/test/lib/testUtils.ts

```diff
@@ -298,6 +299,26 @@ export async function sendMessageToMultipleRecipients(transportServices: Transpo
 
     return response.value;
 }
+export async function failToSendMessageToMultipleRecipients(
```

Comment:
it's simpler to refactor the existing `sendMessageToMultipleRecipients` to work even when failing